src/main/resources/static/css/app.css

```diff
@@ -1,143 +0,0 @@
-body {
```

Comment:
This makes me nervous...I'm wondering where all my precious CSS went...but I'm going to patiently wait to see it re-appear further down.